src/viur/core/modules/user.py

```diff
@@ -321,30 +330,28 @@ def login(self, *, name: str | None = None, password: str | None = None, **kwarg
         self.loginRateLimit.assertQuotaIsAvailable()
 
         name = name.lower().strip()
-        query = db.Query(self._user_module.viewSkel().kindName)
-        user_entry = query.filter("name.idx >=", name).getEntry() or {}  # might find another user; always keep a dict
+        query = self._user_module.baseSkel().all()
+        user_skel = query.filter("name.idx >=", name).getSkel() or {}  # might find another user; always keep a dict
 
-        password_data = user_entry.get("password") or {}
+        password_data = user_skel.dbEntity.get("password") or {}
```

Comment:
What if `getSkel` does not return a result then you get in here 
`AttributeError: 'dict' object has no attribute 'dbEntity'`

Reply:
I've resolved this differently.